include/npcomp/Dialect/ATen/ATenDialect.h

```diff
@@ -0,0 +1,130 @@
+// (c) Copyright 2019 Xilinx Inc. All Rights Reserved.
```

Comment:
forgive my ignorance, but having the "copyright xilinx" here is unusual for code contributed to LLVM umbrella projects. Just doublechecking that this is intended to be here.